lib/TerminalDisplay.cpp

```diff
@@ -431,12 +440,14 @@ TerminalDisplay::TerminalDisplay(QWidget *parent)
   setLayout( _gridLayout );
 
   new AutoScrollHandler(this);
+  autoHideMouseAfter(1000);
```

Comment:
Did you add this for testing? By default it should be disabled.

Reply:
Yes, as mention in the PR this is for demo/testing since this can currently not be wired up to eg. qterminal.
It's not supposed to be part of the final merge.